.gitignore

```diff
@@ -103,10 +103,7 @@ linux-build
 win32-build
 qa/pull-tester/run-bitcoind-for-test.sh
 qa/pull-tester/tests_config.py
-qa/pull-tester/test.*/*
-qa/tmp
 qa/cache/*
-share/BitcoindComparisonTool.jar
```

Comment:
There was a component used for testing that was based on bitcoinj. It was finally removed (in favor of the internal Python based testing framework) in #8504